.circleci/config.yml

```diff
@@ -107,6 +107,7 @@ jobs:
           name: Install dependencies
           command: |
             source env/bin/activate
+            pip install pip==18.1
```

Comment:
*not a blocker*
💅 Semantically, I think this command belongs under "Setup Python environment" rather than "Install dependencies".

Reply:
You're right, I moved it now